apps/docs/src/common/pages/contribute/design-system-working-group.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+import { menu } from '../contribute';
+
+export const title = 'Design system working group';
+const description = 'The governance of the design system';
+const section = 'Contribute';
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <Helmet>
+      <title>{title} - Home Office Design System</title>
+      <meta name="description" content={description} />
+      <meta name="og:title" content={title} />
+      <meta name="og:description" content={description} />
+      <meta name="og:article:section" content={section} />
+    </Helmet>
+    <div className="govuk-grid-column-one-quarter">
+      {menu}
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+      <h1>
+        {title}
```

Comment:
```suggestion
        <span className="caption">{section}</span>
        {title}
```
On the other sub-pages we include a caption for the overall section. (So 'Contribute' in this case.)